ynr/apps/bulk_adding/helpers.py

```diff
@@ -0,0 +1,93 @@
+from popolo.models import Membership, Person
+
+from candidates.models import (LoggedAction, MembershipExtra, PersonExtra,
+                               raise_if_unsafe_to_delete)
+from candidates.models.auth import check_creation_allowed
+from candidates.views.version_data import get_change_metadata, get_client_ip
+
+
+def add_person(request, person_data):
+    # TODO Move this out of the view layer
+    person = Person.objects.create(name=person_data['name'])
+    person_extra = PersonExtra.objects.create(base=person)
+
+    check_creation_allowed(
```

Comment:
Another potentially silly question then: Should we be calling `check_creation_allowed()` before persisting the objects to the DB, or does `check_creation_allowed()` not do quite like what I think it sounds like it does..?

Reply:
It's s good question! 

I think `check_creation_allowed` could do with renaming, as it's really check_**candidacy**_creation_allowed – that is, we're making sure that a person can be added as a member of a (post, election).

AFIK there isn't a limit to people creating people at any time.